README.md

```diff
@@ -1,6 +1,7 @@
 
```

Comment:
There seems to be an empty line before the logo.

Reply:
Seems ok? Doesn't look like it changes the appearance